README.md

```diff
@@ -1,8 +1,13 @@
-ELK autopilot pattern
+Autopilot Pattern ELK
 ==========
 
 *[Autopilot pattern](http://autopilotpattern.io/) implementation of ELK*
 
+[![DockerPulls](https://img.shields.io/docker/pulls/autopilotpattern/mysql.svg)](https://registry.hub.docker.com/u/autopilotpattern/mysql/)
```

Comment:
Oh, that's because there are two images, not an autopilotpattern/elk image: https://hub.docker.com/r/autopilotpattern/kibana/ and https://hub.docker.com/r/autopilotpattern/logstash/